.taskcluster.yml

```diff
@@ -121,6 +122,7 @@ tasks:
           image: python:3-alpine
           env:
             TASKCLUSTER_SECRET: project/relman/taskboot/deploy
+            TASKCLUSTER_ROOT_URL: '${taskcluster_root_url}'
```

Comment:
@djmitche is this a good use case of this new context?

Reply:
I think i have to update task-boot to make it work: i'm using `https://taskcluster.net` as a default `rootUrl`: https://github.com/mozilla/task-boot/blob/master/taskboot/config.py#L52

Reply:
Made a PR for that : https://github.com/mozilla/task-boot/pull/77

@djmitche If you could look over it ... (just a default url usage)

Reply:
For modern workers, it's redundant -- they'll set that value themselves.  But it doesn't hurt.

Reply:
So we should probably remove it as it breaks our taskcluster_yml_validator CI check

Reply:
ah, that tool might need an update to include that context property..

Reply:
Done: https://github.com/marco-c/taskcluster_yml_validator/commit/f56f33c8eec484ccd2e8eb23879a31dee65bf85f.

Reply:
(anyway, better to remove it if it's redundant)

Reply:
ah, good to know. removed!